commands/application.angular.js

```diff
@@ -1,10 +1,16 @@
 const path = require('path');
-const runNpxCommand = require('../utility/run-npx-command');
+const fs = require('fs');
+const runCommand = require('../utility/run-command');
 
 function runSchematicCommand(schematicCommand, options, evaluatingOptions) {
     const collectionName = 'devextreme-schematics';
+    const collectionPackageJsonPath = path.join(process.cwd(), `node_modules/${collectionName}/package.json`)
     let collectionPath = collectionName;
 
+    if(!fs.existsSync(collectionPackageJsonPath)) {
+        runCommand('npm', ['install', collectionName]);
```

Comment:
Should not we evaluate the next command only after this command will be finished?

Reply:
Commands will be executed sequentially. But I don't know how it works on different platforms. I think it's better to fix it.